src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/TimeEntriesTest.kt

```diff
@@ -55,4 +61,32 @@ class TimeEntriesTest: AbstractTest() {
             }
         }
     }
+
+    /**
+     * Tests /v1/ForecastTimeEntriesDelete/webhook endpoint
+     */
+    @Test
+    fun testForecastTimeEntriesDeleteWebhook() {
+        createTestBuilder().use { testBuilder ->
+            testBuilder.manager.synchronization.synchronizeEntries(
+                    before = null,
+                    after = getThirtyDaysAgo().toString()
+            )
+
```

Comment:
Should there be an assertion that the timeEntry exists before it is deleted?

Reply:
Yes there should.